DNN Platform/Dnn.AuthServices.Jwt/Dnn.Jwt.dnn

```diff
@@ -31,12 +31,14 @@
         </component>
         <component type="Script">
           <scripts>
-            <basePath>DesktopModules\AuthenticationServices\JWTAuth\Data\Scripts</basePath>
+            <basePath>DesktopModules\AuthenticationServices\JWTAuth</basePath>
```

Comment:
Oh, how is this related to build warnings? 🙂

Reply:
The build scripts will try to copy these files from project directory(without basePath defined), and throw out warnings because these files aren't there.